sc-poc/managers/admin/handler_login.go

```diff
@@ -0,0 +1,57 @@
+package admin
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"time"
+
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/spacecloud-io/space-cloud/utils"
+)
+
+type Login struct {
+	Secret   string `json:"secret"`
+	Username string `json:"username"`
+	Password string `json:"password"`
```

Comment:
I suggest you don't pass sensitive information like this in the handler. Why don't you get an instance of the `admin` app. The admin app can pull these parameters directly from viper.